internal/database/dbconn/global.go

```diff
@@ -0,0 +1,25 @@
+package dbconn
+
+import "database/sql"
+
+var (
+	// Global is the global DB connection.
+	// Only use this after a call to SetupGlobalConnection.
+	Global *sql.DB
```

Comment:
```suggestion
	// Only use this after a call to SetupGlobalConnection.
	//
	// Deprecated: Pass on DB interface as an argument instead.
	Global *sql.DB
```

Since you've touched this file, would be bonus to add this deprecation notice.